src/mumble/VoiceRecorderDialog.cpp

```diff
@@ -106,6 +116,22 @@ void VoiceRecorderDialog::closeEvent(QCloseEvent *evt) {
 	QDialog::closeEvent(evt);
 }
 
+void VoiceRecorderDialog::on_qrbDownmix_clicked() {
+	qgbOutput->setEnabled(true);
```

Comment:
I don't see a corresponding code path that would disable `qgbOutput` again (except via reset) - is this intentional?

Reply:
Pressing the record button disables the entire dialog box (of which qgbOutput is a child) apart from the stop button. I've tested all possible input scenarios and have confirmed that each case produces expected output (e.g qgbOutput is enabled only for downmix, multichannel, and multichannel + transport options)